.scala-steward.conf

```diff
@@ -2,7 +2,6 @@ updates.ignore = [
     { groupId = "com.google.apis", artifactId="google-api-services-bigquery" },
     { groupId = "org.apache.avro", artifactId="avro" },
     { groupId = "com.google.guava", artifactId="guava" },
-    { groupId = "com.spotify" },
```

Comment:
I suggest removing this so we get scio upgrade prs, even though we'll still have to manually check our build.sbt file for other dependency conflicts before merging them https://spotify.slack.com/archives/C0QC1CLGK/p1694198975357739?thread_ts=1694190469.037639&cid=C0QC1CLGK